packages/@okta/vuepress-site/code/rest/index.md

```diff
@@ -21,19 +21,26 @@ You need a free Okta developer edition org to get started. Don't have one? [Crea
-
-7. Click the gear icon to the right of `{yourOktaDomain}` and select your org to replace or add these values:
-    * Rename your environment to something that you recognize, for example: `My Org`.
-    * `url`: Replace the example value with your org's full URL: For example, `https://mytestorg.oktapreview.com`. Make sure that you don't include `-admin` in the subdomain.
-    * `apikey`: Enter the API token that you created earlier, for example: `00LzMWxMq_0sdErHy9Jf1sijEGexYZlsdGr9a4QjkS`.
-
-8. Click **Update** to save your changes and then close the **Manage Environments** window.
+3. Start Postman if it's not open already.
+4. At the upper-left corner, click **Import**.
+![Postman window](/img/postman_import_button.png "Displays an arrow pointing to the Import button at the upper-left corner of the Postman window")
+5. In the **Import** dialog box, click **Link**, then paste this link: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment` in the **Enter a URL** box and click **Continue**.
+![Postman Import dialog](/img/postman_import_dialog.png "Displays an arrow pointing to the Link tab in the Import dialog box")
+6. Under **Confirm your import**, click **Import**. ![Postman Import dialog for confirming import](/img/postman_import_dialog_confirm.png "Displays an arrow pointing to the Import button in the lower-right corner of the Import dialog box")
+A notification confirms that the import is complete.
+7. At the upper-right corner, click the box that lists the environments, and then select `{yourOktaDomain}` from the list. (The initial value is **No Environment** if you just installed Postman.) ![Postman environment list box](/img/postman_environment_list.png "Displays an arrow pointing to the box in the upper-right corner of the window that contains environments for use with Postman")
+8. At the upper-right corner, next to `{yourOktaDomain}`, click the **eye** symbol (Environment quick look). ![Postman environment quick look button](/img/postman_environment_quick_look.png "Displays an arrow pointing to the eye button in the upper-right corner of the Postman window")
+9. In the upper-right corner of the {yourOktaDomain} dialog box, click **Edit**. 
+![Postman environment quick look edit link](/img/postman_environment_quick_look_edit.png "Displays an arrow pointing to the edit link in the top-right corner of the {yourOktaDomain} dialog box")
+10. In the **MANAGE ENVIRONMENTS** dialog box, do the following:
+    * In the **Environment Name** box, delete the placeholder text and name your environment to something you recognize, for example: `John's Okta Org`.
+    * In the table row for the **VARIABLE** `url`, on the **INITIAL VALUE** and **CURRENT VALUE** columns, replace the placeholder text with your org's full URL. For example, `https://dev-1234567.okta.com`. Remember to _remove_ the `-admin` part in the subdomain.
```

Comment:
```suggestion
    * For the `url` variable, in the **INITIAL VALUE** and **CURRENT VALUE** columns, replace the placeholder text with your org's full URL, for example: `https://dev-1234567.okta.com`. Remember to remove the `-admin` part of your subdomain.
```

Move the Note from the bottom of this section here. You didn't include the note in your PR, but it's out of place now and works nicely here with the "Remember to remove.." sentence.

Note: https://{yourOktaDomain} is different from your admin URL. Don’t include -admin in the value. When you copy your Okta domain from the Developer Console, you can find the correct value in the upper-right corner of the dashboard.